wire/msgaddrv2.go

```diff
@@ -163,11 +181,25 @@ func writeNetAddressV2(op string, w io.Writer, pver uint32, na *NetAddressV2) er
 // maxNetAddressPayloadV2 returns the max payload size for an address manager
 // network address based on the protocol version.
 func maxNetAddressPayloadV2(pver uint32) uint32 {
-	const timestampSize = 8
-	const servicesSize = 8
-	const addressTypeSize = 1
-	const maxAddressSize = 16
-	const portSize = 2
+	if pver < RelayTORv3Version {
+		const (
```

Comment:
These could all go outside of the if and be reused since all of them are the same except `maxAddressSize`.